apps/src/applab/applab.js

```diff
@@ -62,6 +59,9 @@ import {
   actions as jsDebugger,
 } from '../lib/tools/jsdebugger/redux';
 import JavaScriptModeErrorHandler from '../JavaScriptModeErrorHandler';
+import connectToMakerBoard from '../lib/kits/maker/connectToMakerBoard';
+import * as makerCommands from '../lib/kits/maker/commands';
+import * as makerDropletConfig from '../lib/kits/maker/dropletConfig';
```

Comment:
yeah, that's what was decided. But I think the reasoning for it might have been a little off. From what I remember, people didn't want their editors to pull up a zillion different `index.js` files when they typed `index.js`. But we actually already have this problem with `redux.js`, of which there are many. IMO, it's not a big deal to pick among a set of files with the same name, and the way that it makes "public" apis more explicit is advantageous.